plone/namedfile/z3c-blobfile.zcml

```diff
@@ -56,6 +56,12 @@
       factory=".storages.PDataStorable"
       />
 
+  <utility
+      factory=".storages.Zope2FileUploadStorable"
+      provides=".interfaces.IStorage"
+      name="ZPublisher.HTTPRequest.FileUpload"
+      />
```

Comment:
@gforcada My only doubt is: why isn't this ZCML conflicting with https://github.com/plone/plone.app.z3cform/blob/master/plone/app/z3cform/configure.zcml#L116?